### DISCUSSION TITLE
Editable not showing in both skeleton and demo pimcore fresh setup in 10.0

### DISCUSSION THREAD

**Discussion by baladpi** [pimcore/pimcore#9805] [General]
![newpim-loc-Pimcore](https://user-images.githubusercontent.com/8745583/126272239-4e16e0ba-d4ff-473e-a88d-455ebd31e977.png)

My code :

{% extends 'layout.html.twig' %}
{% block content %}
<h1>{{ pimcore_input("headline", {"width": 540}) }}</h1>

<div class="product-info">
    {% if editmode %}
    <p>asasa</p>
        {{ pimcore_relation("product") }}
        <p>asasa</p>
    {% else %}
        {% set product = pimcore_relation("product").element %} 
        {% if product %} 
            <h2>{{ product.name }}</h2>
            <div class="content">
                {{ product.description }}
            </div>
        {% endif %}
    {% endif %}
</div>{% endblock %}

**Comment by dvesh3** [comment]
sorry, it works on https://demo.pimcore.fun (which refreshes in every 2 hrs).

**Reply by baladpi** [reply]
Thanks for your response, anything needs to be check or install any extensions

**Comment by baladpi** [comment]
Actually, am not getting any errors on console too
